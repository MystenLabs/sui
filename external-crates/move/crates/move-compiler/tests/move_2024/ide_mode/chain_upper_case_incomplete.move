module 0x42::m1 {
    public fun bar() {}

    public fun foo() {
        B
    }
}
