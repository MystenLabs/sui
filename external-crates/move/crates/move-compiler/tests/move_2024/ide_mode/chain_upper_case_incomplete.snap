error[E03006]: unexpected name in this position
  ┌─ tests/move_2024/ide_mode/chain_upper_case_incomplete.move:5:9
  │
5 │         B
  │         ^ Expected a type, function, or constant in this position, not an address

